docs/participate/lite-phase.md

```diff
@@ -4,43 +4,49 @@ sidebar_position: 0
 
 # Lite Node Phase
 
-We are thrilled to announce that Early Mint for our Snapshotter Lite Node is live! This is an opportunity for everyone to participate in our Composable Data Network. You can now mint up to 10 slots and run your Snapshotter Lite Nodes.
+We are excited to announce the launch of the public Mint for our Snapshotter Lite Node, now live and open to all! This marks a significant milestone in our Composable Data Network, offering everyone the chance to mint node slots and operate their own Snapshotter Lite Nodes.
 
-Our pre-mint successfully concluded on February 4th, 2024.
-For more details, please visit our blog post: [First Data Infrastructure Mint on Polygon](https://blog.powerloom.io/first-data-infrastructure-mint-on-polygon-launching-over-10-000-data-nodes/).
+### Key Milestones in Our Node Mint Journey:
+- ✅ Pre-Mint Kickoff: January 25th, 2024
+- ✅ Pre-Mint Closure: February 4th, 2024
+- ✅ Pre-Mint Selection: February 6th, 2024
+- ✅ Early Mint (Exclusive): February 7th, 2024 
+- ⏳ Public Mint Opens: February 9th, 2024 (ongoing)
 
-:::info
-Important Announcement: Our Testnet is gearing up for launch shortly. In the interim, we invite developers to experiment with running our Snapshotter node.
 
-While currently operating the node does not yield rewards, this is an excellent opportunity to simulate and test the Snapshotter node, providing valuable insights into its functionality and operation.
-:::
+For detailed information, please refer to our Snapshotter Node Mint documentation: [Snapshotter Node Mint Documentation](https://powerloom.network/mint-docs/)
+
+For additional insights, visit our blog post: [First Data Infrastructure Mint on Polygon](https://blog.powerloom.io/first-data-infrastructure-mint-on-polygon-launching-over-10-000-data-nodes/).
 
-## Introducing Snapshotter Lite Nodes
+:::info Important Announcement
+ As we prepare for our Testnet launch, we invite developers to explore and experiment with operating our Snapshotter node. Currently, operating the node does not provide rewards, but it offers a valuable opportunity to simulate, test, and gain insights into the Snapshotter node's functionality and operation.
+:::
 
-After acquiring node slots, you can operate our Snapshotter Lite Node. This version is more streamlined than the full Snapshotter Node, requiring only a Python environment and considerably fewer resources. It's ideal for simpler scenarios that don't need data aggregation.
+## What Are Snapshotter Lite Nodes?
 
-## How Snapshotter Lite Node Functions within the PowerLoom Ecosystem
+Once you have acquired your node slots, you can begin operating our Snapshotter Lite Node. This streamlined version of the full Snapshotter Node requires only a Python environment and significantly fewer resources, making it ideal for less complex scenarios that do not require data aggregation.
```

Comment:
making it ideal for lightweight nodes that do not need to contribute to heavyweight computations on aggregations and other data compositons